packages/fluentui/docs/src/routes.tsx

```diff
@@ -105,13 +107,15 @@ const VirtualizedTablePrototype = React.lazy(() =>
 const FormValidationPrototype = React.lazy(() =>
   import(/* webpackChunkName: "prototypes" */ './prototypes/FormValidation'),
 );
+const RosterPrototype = React.lazy(() => import(/* webpackChunkName: "prototypes" */ './prototypes/Roster'));
 
 const Routes = () => (
   <React.Suspense fallback="Loading...">
     {/* Remove trailing slash */}
     <BrowserRouter basename={__BASENAME__ === '/' ? __BASENAME__ : __BASENAME__.slice(0, -1)}>
       <Switch>
         <Route exact path="/maximize/:exampleName/:rtl?" component={ExternalExampleLayout} />
+        <Route exact path="/maximize-compiled/:exampleName/:rtl?" component={PerfExampleLayout} />
```

Comment:
@layershifter recently changed `ExternalExampleLayout` to render compiled examples.
How do `/maximize` and `/maximize-compiled` differ? Can we converge to just one?

Reply:
Changed the code so that the performace popout is using maximize and it seems to be working.